modules/keepnick.cpp

```diff
@@ -174,7 +174,7 @@ class CKeepNickMod : public CModule {
                 return HALT;
             }
 
-            PutModule("Unable to obtain nick: " + numeric.GetParam(2) + ": " + numeric.GetParam(3));
+            PutModule("Unable to obtain nick: " + numeric.GetParam(1) + ": " + numeric.GetParam(2));
```

Comment:
I just took your test from your PR (you deleted that test), and made it work. I don't know the actual format of 435 and 447. The test now matches the behavior.
That's one of the points of tests, after all.
If the format is wrong, may you suggest fixes to test and to this string?